.github/workflows/wheels.yml

```diff
@@ -154,7 +154,7 @@ jobs:
         run: |
           python -VV
           # Install the local wheel
-          pip install dist/tskit-*.whl
```

Comment:
try ``python -m pip install`` - looks to me like it's a mixing-python-versions issue.